packages/api/src/controllers/asset.ts

```diff
@@ -654,6 +674,14 @@ app.get("/", authorizer({}), async (req, res) => {
     query.push(sql`asset.data->>'deleted' IS NULL`);
   }
 
+  if (projectId) {
```

Comment:
If we filter by `projectId`, do we also need to filter by `userId` in the line 683?

Reply:
Yes, don't we? We're trying to get all assets beloniging to a specific userId broken down by a specific projectId.

Reply:
Doesn't one project belong to one user only? Then, filtering by projectId should be enough.

Reply:
I don't think projectId should be its own queryparam like this, we can add it to the `fieldsMap` instead. I also think we should treat it similar to userId in that only admins are allowed to use it. For non-admins we should just take the projectId associated with the api key being used.